### PR TITLE
Added Dapr-RetroPOS external sample

### PR DESCRIPTION
**Pull request by robece**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,9 @@ If you are new to Dapr, you may want to review following resources first:
 
 ## External samples
 
->This section will include external links to Dapr related samples, located outside the Dapr Samples repository.
+| Sample | Details | 
+|------|-------|
+| [Dapr RetroPOS](https://github.com/robece/dapr-retropos) | Dapr Retro Point of Sales (Backend Workflows) is an event-driven microservices based architecture for Contoso Ltd., which will enable scalability with a plugable design for the next generation of cloud point of sales. |
```
Comment (mchmarny) [inline README.md:32]: Consider something less sales maybe like: 

> Dapr Retro Point of Sales is a sample of backend workflow based on microservices....

I would also recommend leaving out company name and focus more on how users would benefit from using reviewing this sample.

**Review by mchmarny — CHANGES_REQUESTED** [review]
See comments

File: README.md (hunk 1 above)
Comment (robece) [reply README.md]: Thanks Mark, I updated the details to keep it simple without the company name.

**Review by mchmarny — APPROVED** [review]
/lgtm

**Comment by orizohar** [comment]
Thanks @robece !

**Pull request merged**